pysd/py_backend/vensim/vensim2py.py

```diff
@@ -602,15 +603,18 @@ def parse_general_expression(element, namespace=None, subscript_dict=None, macro
     array = (number _ ("," / ";")? _)+ !~r"."  # negative lookahead for anything other than an array
     number = ~r"\d+\.?\d*(e[+-]\d+)?"
 
-    id = ~r"(%(ids)s)"I
```

Comment:
Makes identifiers is not restricted for correct handling unicode chars. Don't have any ideas why strict implementation of this rule don't work in Python 2.7